SampleApps/FRExample/FRExample/ViewController.swift

```diff
@@ -214,14 +215,41 @@ class ViewController: UIViewController, ErrorAlertShowing {
         */
         
         // Start SDK
+      if !useDiscoveryURL {
+        // use the Config Plist file
         do {
-            try FRAuth.start()
-            self.displayLog("FRAuth SDK started using \(FRAuth.configPlistFileName).plist.")
+          try FRAuth.start()
+          self.displayLog("FRAuth SDK started using \(FRAuth.configPlistFileName).plist.")
         }
         catch {
-            self.displayLog(String(describing: error))
+          self.displayLog(String(describing: error))
         }
-        
+      } else {
+        // use the discovery URL
+        if #available(iOS 13.0, *) {
+          Task {
+            do {
+              let config =
+              ["forgerock_oauth_client_id":"c12743f9-08e8-4420-a624-71bbb08e9fe1",
+               "forgerock_oauth_redirect_uri": "org.forgerock.demo://oauth2redirect",
+               "forgerock_oauth_scope" : "openid profile email address revoke",
+               "forgerock_ssl_pinning_public_key_hashes":["nz2Yw6N2Pv7ctMxG+lSMTQUy0DsBDF6+NHHkvmg4G6s="]]
+
+              let discoveryURL = "https://auth.pingone.ca/02fb4743-189a-4bc7-9d6c-a919edfe6447/as/.well-known/openid-configuration"
```

Comment:
Maybe on the FRExample we leave this configurable so that devs can add their own configuration. We could add a UI component that they would set those details

Reply:
I changed it to a PLACEHOLDER. If ran in `useDiscoveryURL = true` mode, it will throw an error if Placeholders are not replaced.